docs/azure-config.md

```diff
@@ -120,47 +116,34 @@ To integrate Velero with Azure, you must create an Velero-specific [service prin
+Now you need to create a file that contains all the environment variables you just set. The command looks like the following:
 
 ```bash
-kubectl apply -f config/common/00-prereqs.yaml
+cat << EOF  > ./credentials-velero \
+AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID} \
+AZURE_TENANT_ID=${AZURE_TENANT_ID} \
+AZURE_CLIENT_ID=${AZURE_CLIENT_ID} \
+AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET} \
+AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP}
+EOF
 ```
 
-Now you need to create a Secret that contains all the environment variables you just set. The command looks like the following:
+## Install and start Velero
+
+Install Velero, including all prerequisites, into the cluster and start the deployment. This will create a namespace called `velero`, and place a deployment named `velero` in it.
 
 ```bash
-kubectl create secret generic cloud-credentials \
-    --namespace <VELERO_NAMESPACE> \
-    --from-literal AZURE_SUBSCRIPTION_ID=${AZURE_SUBSCRIPTION_ID} \
-    --from-literal AZURE_TENANT_ID=${AZURE_TENANT_ID} \
-    --from-literal AZURE_CLIENT_ID=${AZURE_CLIENT_ID} \
-    --from-literal AZURE_CLIENT_SECRET=${AZURE_CLIENT_SECRET} \
-    --from-literal AZURE_RESOURCE_GROUP=${AZURE_RESOURCE_GROUP}
+velero install \
+    --bucket <YOUR_BLOB_CONTAINER> \
+    --provider azure \
+    --secret-file ./credentials-velero \
+    --backup-location-config resourceGroup=<YOUR_STORAGE_RESOURCE_GROUP>,storageAccount=<YOUR_STORAGE_ACCOUNT> \
+    --snapshot-location-config timeout=<YOUR_TIMEOUT>
 ```
 
-Now that you have your Azure credentials stored in a Secret, you need to replace some placeholder values in the template files. Specifically, you need to change the following:
-
-* In file `config/azure/05-backupstoragelocation.yaml`:
-
-  * Replace `<YOUR_BLOB_CONTAINER>`, `<YOUR_STORAGE_RESOURCE_GROUP>`, and `<YOUR_STORAGE_ACCOUNT>`. See the [BackupStorageLocation definition][21] for details.
-
-* In file `config/azure/06-volumesnapshotlocation.yaml`:
-
-  * Replace `<YOUR_TIMEOUT>`. See the [VolumeSnapshotLocation definition][8] for details.
-
-* (Optional, use only if you need to specify multiple volume snapshot locations) In `config/azure/00-deployment.yaml`:
-
-  * Uncomment the `--default-volume-snapshot-locations` and replace provider locations with the values for your environment.
-
-## Start the server
-
-In the root of your Velero directory, run:
+Additionally, you can specify `--use-restic` to enable restic support, and `--wait` to wait for the deployment to be ready.
 
-  ```bash
-  kubectl apply -f config/azure/
-  ```
+For more complex installation needs (such as changing the namespace), use either the Helm chart, or add `--dry-run -o yaml` options for generating the YAML representation for the installation.
```

Comment:
We should add a note about the nginx example at the end here, for parity with the other 2.